crates/rolldown/tests/esbuild/loader/auto_detect_mime_type_from_extension/artifacts.snap

```diff
@@ -1,5 +1,5 @@
 ---
-source: crates/rolldown/tests/common/case.rs
+source: crates/rolldown_testing/src/case/case.rs
```

Comment:
@IWANABETHATGUY I also meet it if snapshot content changes. Could you see see?

Reply:
It is expected, the new path is correct, this is because we moved the `case` to another crate, but `cargo insta` only update the snapshot when the content is changed.